.insomnia/Request/req_5e2d111bc0b04c2190720ff43414adc2.yml

```diff
@@ -0,0 +1,23 @@
+_id: req_5e2d111bc0b04c2190720ff43414adc2
+type: Request
+parentId: fld_eeb62a531f8441478a6d2f536021400d
+modified: 1659130784956
+created: 1659130480699
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/searches/new"
+name: Searches Pre-Signed URL
+description: Endpoint for generating a pre-signed URL to upload searches to S3.
```

Comment:
We need to document the validity of the token and mention that any number of files can be uploaded under this prefix as well as other considerations like what response codes are returned from S3 when it becomes invalid so that clients know when to regenerate them.

Reply:
Added more documentation, lmk if I'm missing anything.